src/components/ViewRoutingSlip/RoutingSlipInfo.vue

```diff
@@ -1,11 +1,144 @@
 <template>
   <div>
-    <h3>01.Routing Slip Information</h3>
+    <header class="d-flex flex-column mb-0">
+      <h3>01.Routing Slip Information</h3>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+    </header>
+    <v-card class="pl-5 py-2">
+      <v-card-text>
+        <v-row no-gutters>
+          <v-col class="col-12 col-sm-10 ">
+            <v-row no-gutters>
+              <v-col class="col-6 col-sm-3 font-weight-bold">
+                Routing Slip - Unique ID
+              </v-col>
+              <v-col class="col-6 col-sm-9">
+                {{ routinSlipDetails.number }}
+              </v-col>
+            </v-row>
+
+            <v-row>
+              <v-col class="col-6 col-sm-3 font-weight-bold">
+                Date
+              </v-col>
+
+              <v-col class="col-6 col-sm-9">
+                {{
+                  formatDisplayDate(
+                    routinSlipDetails.routingSlipDate,
+                    'MMM DD, YYYY'
+                  )
+                }}
+              </v-col>
+            </v-row>
+
+            <v-row v-if="!editMode">
+              <v-col class="col-6 col-sm-3 font-weight-bold">
+                Status
+              </v-col>
+              <v-col class="col-6 col-sm-9">
+                <span
+                  :class="colors(routinSlipDetails.status)"
+                  class="slip-status"
+                  >{{ getStatusLabel(routinSlipDetails.status) }}</span
+                >
+              </v-col>
+            </v-row>
+            <!-- show only on edit -->
+            <v-row v-else>
+              <v-col class="col-6 col-sm-3 font-weight-bold">
+                Status
+              </v-col>
+              <v-col class="col-6 col-sm-9">
+                <statusList v-model="currentStatus"></statusList>
+              </v-col>
+            </v-row>
+
+            <v-row>
+              <v-col class="col-6 col-sm-3 font-weight-bold">
+                Name of Person Submitting
+              </v-col>
+              <v-col class="col-6 col-sm-9">
+                {{ routinSlipDetails.createdName }}
+              </v-col>
+            </v-row>
+          </v-col>
+
+          <v-col class="col-1 col-sm-2 d-flex justify-end pr-5"
```

Comment:
need to update permission in this component. will do once we implement v-can in fas like sbc-auth